### DISCUSSION TITLE
Sorting posts by date, etc.

### DISCUSSION THREAD

**Discussion by Zero3K** [bluesky-social/atproto#1347] [Bluesky Lexicons]
It would be nice if Bluesky had an option to sort posts by date, etc.

**Comment by Zero3K** [comment]
Any thoughts on this?

**Reply by bnewbold** [reply]
Could you write a paragraph or two explaining what you are trying to achieve, and what you have already tried? Which posts, in what context, what software is involved, etc.

**Reply by Zero3K** [reply]
I am not good at writing paragraphs. And what do you mean by which software is involved? I am wanting the feeds to be sorted by date (newest to oldest).

**Comment by Zero3K** [comment]
At least make it be like how Twitter does it (which is newest to oldest).

**Reply by Zero3K** [reply]
Well?
